sdk/PrebidServerAdapter/PBServerAdapter.m

```diff
@@ -140,6 +142,21 @@ - (NSDictionary *)requestBodyForAdUnits:(NSArray<PBAdUnit *> *)adUnits {
     return [requestDict copy];
 }
 
+- (NSURL *)pbServerUrl {
+    NSURL *url;
+    switch ([[PBServerHost sharedInstance] pbsHost]) {
+        case PBSHostRubicon:
+            url = [NSURL URLWithString:kRPPrebidServerUrl];
+            break;
+        case PBSHostAppNexus:
+        default:
```

Comment:
Based on conversation with Pieter I'll change the default to throw an error, making explicitly setting the host required.